integration/map.go

```diff
@@ -85,7 +86,7 @@ func RunMapIntegration(ctx context.Context, mapID int64, client trillian.Trillia
 
 	// Check values
 	// Mix up the ordering of requests
-	h := merkle.NewMapHasher(testonly.Hasher)
+	h := rfc6962.DefaultHasher
```

Comment:
I'd have thought you specifically don't want an RFC hasher for a map test as it's never going to use the depth stuff. We want to minimize dependencies on CT specifics.